accountsdb-plugin-manager/src/accountsdb_plugin_service.rs

```diff
@@ -156,12 +155,12 @@ impl AccountsDbPluginService {
             )));
         }
 
-        let result: serde_json::Value = match serde_json::from_str(&contents) {
+        let result: serde_json::Value = match json5::from_str(&contents) {
             Ok(value) => value,
             Err(err) => {
                 return Err(AccountsdbPluginServiceError::InvalidConfigFileFormat(
                     format!(
-                        "The config file {:?} is not in a valid Json format, error: {:?}",
+                        "The config file {:?} is not in a valid Json5 format, error: {:?}",
```

Comment:
Can you update the doc in https://github.com/solana-labs/solana/blob/master/docs/src/developing/plugins/accountsdb_plugin.md to reflect it uses JSON5 format.